archetypes/archetypes/src/main/archetype/mp/custom/custom-mp.xml

```diff
@@ -44,6 +44,8 @@
         <model>
             <value key="helidon-test">true</value>
             <value key="mainClass" if="${jpms}">${package}.Main</value>
+            <value key="idea-main-class" if="${jpms}">${package}.Main</value>
```

Comment:
Can we have a mustache if-else condition for this instead ?

Reply:
Much better, thanks !